docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -61,6 +62,16 @@ \subsection{Environment}
 \wss{Hardware and software}
 
 \section{Goals}
+\begin{itemize}
+\subsection{Work Orders}
```

Comment:
I don't think making this entire section and a list is the way to go. Since you define the subsections inside the list, I think it makes the indentation look weird.